plugin/geoip/geoip_test.go

```diff
@@ -37,6 +37,11 @@ func TestMetadata(t *testing.T) {
 		{"geoip/asn/number", "12345", asnDBPath, "81.2.69.142"},
 		{"geoip/asn/org", "Test ASN Organization", asnDBPath, "81.2.69.142"},
 
+		{"geoip/latitude", "52.2242"},
+		{"geoip/longitude", "0.1315"},
+		{"geoip/timezone", "Europe/London"},
+		{"geoip/postalcode", "CB4"},
+		{"geoip/subdivisions/code", "ENG,CAM"},
```

Comment:
@fr6nco Can you have a look to your conflict resolution in `geoip_test.go` and fix all of the errors?